snap/snapcraft.yaml

```diff
@@ -262,6 +302,16 @@ parts:
     override-pull: ""
     stage-packages:
     - on amd64: [msr-tools]
+    stage:
+    - -usr/share/doc
+    - -usr/share/man
+
+  compile-schemas:
```

Comment:
What is compile-schemas? What does it do?

Reply:
I basically split this out from the `terminal` part in order to not pull in all of libglib.  It's easier to just pull this out of `terminal` and *only* specify the one file that is needed instead of leaving it in `terminal` and having to specify **all** of the files we **don't** want.

At the heart of it, [`glib-compile-schemas`](https://manpages.ubuntu.com/manpages/trusty/en/man1/glib-compile-schemas.1.html) is needed for `xfce4-terminal` to operate correctly.